lib/Importer/Caffe2.cpp

```diff
@@ -454,7 +462,17 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     assert(!transLHS && "Don't support transpose lhs for now.");
     bool transRHS = dict.count("trans_b") && (loadInt(dict["trans_b"]) == 1);
     if (transRHS) {
-      RHS = G_.createTranspose("RHS.transpose", RHS, {1, 0});
+      // The semantic of the transpose in that context is:
+      // swap the last two dimensions.
+      size_t nbDims = RHS.dims().size();
+      std::vector<unsigned_t> shuffle;
+      unsigned_t i;
+      for (i = 0; i < (unsigned_t)nbDims - 2; ++i) {
```

Comment:
Bad things can happen if nbDims is less than 2.

Reply:
Added an assert on that. The C2 specs say that the rank of the inputs must be >= 2.